src/gui/folderstatusmodel.cpp

```diff
@@ -513,9 +489,9 @@ QModelIndex FolderStatusModel::parent(const QModelIndex &child) const
     }
     switch (classify(child)) {
     case RootFolder:
-    case AddButton:
         return QModelIndex();
     case SubFolder:
+        Q_FALLTHROUGH();
```

Comment:
nitpick.
As we require c++17 we should start using `[[fallthrough]];`

Reply:
done